packages/patternfly-4/react-core/package.json

```diff
@@ -73,6 +73,6 @@
   "peerDependencies": {
     "prop-types": "^15.6.1",
     "react": "^16.4.0",
-    "react-dom": "^15.6.2 || ^16.4.0"
+    "react-dom": "^16.4.0"
```

Comment:
This looks like we're forcing apps to upgrade from 15.6.2, which may be a breaking change for some. I don't believe it would hurt leaving 15.6.2 there, unless there is specific functionality required by 16.4.0?